packages/itwinui-css/src/badge/badge.scss

```diff
@@ -27,4 +27,15 @@ $iui-badge-horizontal-padding: calc(var(--iui-size-2xs) * 1.5);
   overflow: hidden;
   text-overflow: ellipsis;
   font-weight: var(--iui-font-weight-semibold);
+
+  &[data-iui-status] {
+    outline: 1px solid;
+    outline-offset: -1px;
```

Comment:
i think it would be better to use border. i know it occupies space and could cause a size inconsistency between status vs other badges, so my suggestion is to add border to all badges like this:
```css
border: 1px solid transparent;
```
and then inside the `iui-badge-status` mixin, you can set `border-color: currentColor`.

as a result, the code is simpler, works in safari and also results in better support for forced-colors.

Reply:
Replaced `outline` with `border`.  It's currently setting `border` on all badges.  Non-status badges have `--border-color: var(--background-color)`, then the status badges define background & border separately.  The border change did create a lot of image changes but `forced-color` support was something i forgot to consider & I think is more important.

Reply:
oh dang. we might want to remove badges from those tile tests - dont want another 100 images changing when we rework badges again

Reply:
How about we test badges within tiles in only 1 sample?  Right now we have badges within 10 tile samples.

Reply:
yeah, feel free to experiment. we can even have multiple Tiles with badges as long as we don't use those same tiles for the more granular tests (hover/focus).

Reply:
Reduced badge usage within tile demos to just 1 new tile.